tests/test_multioutput.py

```diff
@@ -85,6 +85,67 @@ class Data:
     P = 3  # output dimension
     MAXITER = int(15e2)
 
+
+def make_sqrt_data(rng, N, M):
+    return np.array([np.tril(rng.randn(M, M)) for _ in range(N)])  # N x M x M
+
+
+def expand_cov(G, W):
+    ''' G is L x M x M
+        W is L x L
+        Output is LM x LM
```

Comment:
what is this trying to compute? some kind of outer product?

Reply:
yes. the idea is to expand in the 'wasteful' LM * LM shape, covariances saved as L * M * M
(because this is what the dummy mok spits out)